<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QManagerWindow</class>
 <widget class="QDialog" name="QManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>708</width>
    <height>336</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>QManager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <layout class="QHBoxLayout" name="hlayout_1">
     <item>
      <widget class="QPushButton" name="connectPushButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addRulePushButton">
       <property name="text">
        <string>Add rule</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeRulePushButton">
       <property name="text">
        <string>Remove rule</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QHBoxLayout" name="hlayout_6">
     <item>
      <widget class="QTableView" name="rulesTableView"/>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <spacer name="buttons_spacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>300</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="hlayout_buttons">
     <item>
      <widget class="QPushButton" name="closePushButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>closePushButton</sender>
   <signal>clicked()</signal>
   <receiver>QManagerWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>314</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
